packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -825,11 +839,21 @@ export class MgtPerson extends MgtTemplatedComponent {
       oneline: this.isOneLine()
     });
 
+    const activityText = (status: string) => {
+      for (const item in this.strings) {
+        if (item === status) {
+          return this.strings[item] as string;
+        }
+      }
+    };
+
+    const formattedActivity = activityText(activity);
```

Comment:
We don't need to iterate here, we can index and use the nullish coalescing operator. 

```suggestion
    const formattedActivity = this.strings[activity] as string ?? nothing;
```

Here you'll need to update the lit imports to include `nothing`, the nice thing about using nothing as a return type here is that lit has special handling to remove attributes which would be bound to a nothing value.